me.acristoffers.Lachesis.yml

```diff
@@ -0,0 +1,54 @@
+app-id: me.acristoffers.Lachesis
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    YARN_LOGLEVEL: info
+modules:
+  - name: lachesis
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/lachesis/flatpak-node/cache
+    build-commands:
+      - bash build-flatpak.sh
+      - cp -a desktop/dist/linux*unpacked ${FLATPAK_DEST}/main
+      - install -Dm644 icon_16x16.png ${FLATPAK_DEST}/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon_32x32.png ${FLATPAK_DEST}/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon_128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon_256x256.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm644 icon_512x512.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      - install -Dm755 -t /app/bin/ run.sh
+    sources:
+      - type: git
+        url: https://github.com/acristoffers/Lachesis.git
+        branch: master
+        commit: b507c3475d8ec3460df846fa9dc26163ff799369
+      - generated-sources.json
+      # Wrapper to launch the app
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/lachesis "$@"
+      - type: dir
```

Comment:
1. don't use dir
2. we only need ONE size. 128x128. 256x256 can also do it. Everything else is unucessary and should instead be ALL managed upstream.

Reply:
Ok, I'll change it to have only the 256x256 then.